german/Leichen/Schwimmbewegungen/Schwimmbewegungen.md

```diff
@@ -1,7 +1,8 @@
-Es gibt Momente die, obgleich sie nur Minuten andauern, sp�ter Teil unseres Lebens werden als dauerten sie Jahre.
+Es gibt Momente die, obgleich sie nur Minuten andauern, später Teil unseres Lebens werden als dauerten sie Jahre.
 Dies ist ein solcher Moment in eurem Leben.
-Eure ganze Aufmerksamtkeit geht in euren K�rper.
+Eure ganze Aufmerksamtkeit geht in euren Körper.
 Wohlwissend um euer eventuelles Ende, schwimmt ihr wie ihr noch nie geschwommen seit.
 Das schwierigste ist in Mitten der Wellen die Orienturng zu behalten.
 Wenn ihr es nur nah genug an den Strand schafft, findet ihr vielleicht ein Riff oder eine Sandbank,
 eine kleine Verschnaufpause vor der letzten Anstrengung.
+Als ihr dann völlig ausser Atem kommt und schon einen Mund voll Wasser verschluckt habt, schaut ihr noch einmal ans Ufer. Mit dem Wissen dass es gleich zu Ende ist lasst ihr euch treiben. Doch dann spührt ihr wieder plötzlich Boden unter den Füssen.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.